pyo3-ffi/src/cpython/dictobject.rs

```diff
@@ -63,6 +62,14 @@ extern "C" {
     // skipped _PyDictViewObject
     // skipped _PyDictView_New
     // skipped _PyDictView_Intersect
+
+    #[cfg(Py_3_10)]
+    pub fn _PyDict_Contains_KnownHash(
+        op: *mut PyObject,
+        key: *mut PyObject,
+        hash: Py_ssize_t,
```

Comment:
```suggestion
        hash: crate::Py_hash_t,
```

Maybe we should use [`Py_hash_t`](https://docs.rs/pyo3-ffi/latest/pyo3_ffi/type.Py_hash_t.html) for consistency?